### PR TITLE
Add missing definitions to google-libphonenumber

### PR DESCRIPTION
**Pull request by vigbk**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `npm run lint -- package-name` if a `tslint.json` is present.

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Run `tsc` without errors.
- [ ] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header. Base these on the README, *not* on an existing project.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/googlei18n/libphonenumber/blob/master/javascript/i18n/phonenumbers/phonenumberutil.js#L2659
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

**Comment by leonyu** [comment]
:+1:

File: google-libphonenumber/index.d.ts [hunk 1]
```diff
@@ -11,6 +11,21 @@ declare namespace libphonenumber {
...
+        FIXED_LINE_OR_MOBILE,
+        TOLL_FREE,
+        PREMIUM_RATE,
+        SHARED_COST,
+        VOIP,
+        PERSONAL_NUMBER,
+        PAGER,
+        UAN,
+        VOICEMAIL,
+        UNKNOWN
```
Comment (leonyu) [inline google-libphonenumber/index.d.ts]: https://github.com/googlei18n/libphonenumber/blob/master/javascript/i18n/phonenumbers/phonenumberutil.js#L912-L941

UNKNOWN is -1. 

Should probably enumerate all the enums for compatibility.

**Review by leonyu — CHANGES_REQUESTED** [review]

File: google-libphonenumber/index.d.ts (hunk 1 above)
Comment (vigbk) [inline google-libphonenumber/index.d.ts]: Sure. Will update it.

**Comment by leonyu** [comment]
:+1:

**Review by leonyu — APPROVED** [review]

**Comment by vigbk** [comment]
Great. Thanks.

**Pull request merged**